<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>853</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HIT DAQ</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>831</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTextBrowser" name="logWindow"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushRun">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Run!</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushLogging">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Start logging!</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushDisplay">
         <property name="text">
          <string>Show display!</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushLogSettings">
         <property name="text">
          <string>Log settings...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_exit">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>510</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushAnalysing">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>440</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Analysing!</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>853</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDevice">
    <property name="title">
     <string>Device</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionHost_IP"/>
    <addaction name="actionDevices"/>
    <addaction name="actionTrigger_config"/>
   </widget>
   <addaction name="menuDevice"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionHost_IP">
   <property name="text">
    <string>Host IP...</string>
   </property>
  </action>
  <action name="actionDevices">
   <property name="text">
    <string>Devices...</string>
   </property>
  </action>
  <action name="actionTrigger_config">
   <property name="text">
    <string>Trigger config...</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="text">
    <string>Run</string>
   </property>
  </action>
  <action name="actionConnect_Keithley">
   <property name="text">
    <string>Connect Keithley</string>
   </property>
  </action>
  <action name="actionDisconnect_Keithley">
   <property name="text">
    <string>Disconnect Keithley</string>
   </property>
  </action>
  <action name="actionLinearity_test">
   <property name="text">
    <string>Linearity test</string>
   </property>
  </action>
  <action name="actionIntegration_time_scan">
   <property name="text">
    <string>Integration time scan</string>
   </property>
  </action>
  <action name="actionProfile_viewer">
   <property name="text">
    <string>Profile viewer</string>
   </property>
  </action>
  <action name="actionConnect_Stepper">
   <property name="text">
    <string>Connect Stepper</string>
   </property>
  </action>
  <action name="actionDisconnect_Stepper">
   <property name="text">
    <string>Disconnect Stepper</string>
   </property>
  </action>
  <action name="actionBeta_Scanner">
   <property name="text">
    <string>Beta Scanner</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
